documents/generated/asciidoc/index.adoc

```diff
@@ -0,0 +1,378 @@
+= CSAF Validator Service
+team@openapitools.org
+1.0.0
+:toc: left
+:numbered:
+:toclevels: 3
+:source-highlighter: highlightjs
+:keywords: openapi, rest, CSAF Validator Service
+:specDir: 
+:snippetDir: 
+:generator-template: v1 2019-12-20
+:info-url: https://openapi-generator.tech
+:app-name: CSAF Validator Service
+
+[abstract]
+.Abstract
+This is a service to validate documents against the CSAF standard.
+
+
+// markup not found, no include::{specDir}intro.adoc[opts=optional]
+
+
+
+== Endpoints
+
+
+[.Default]
+=== Default
+
+
+[.apiV1TestsGet]
+==== apiV1TestsGet
+
+`GET /api/v1/tests`
+
+Retrieve all tests.
+
+===== Description
+
+This endpoint is intended to be used to discover all available tests.
+
+
+// markup not found, no include::{specDir}api/v1/tests/GET/spec.adoc[opts=optional]
```

Comment:
Is something missing here?

Reply:
These are optional samples that I left out for the moment.